### DISCUSSION TITLE
fastdds 运行 创建  报错  m_pParticipant = DomainParticipantFactory::get_instance()->create_participant(0, participantQos);

### DISCUSSION THREAD

**Discussion by 18871434218** [eProsima/Fast-DDS#4267] [Support]
内部 debug 执行到  part = RTPSDomain::createParticipant(domain_id_, false, rtps_attr, &rtps_listener_);报错
[foonathan::memory] Assertion failure in function push (buildtrees/foonathan-memory/src/v0.7-3-8e7bcb82b7.clean/src/memory_arena.cpp:15): Assertion "block.size >= sizeof(node)" failed.

**Comment by JesusPoderoso** [comment]
Hi @18871434218, thanks for using Fast DDS.
Unfortunately, we cannot help you unless you provide more context information or a reproducer example.

With the information provided, I assume that you wanted to open a support question in Fast DDS repository, so I am moving this ticket to the corresponding forum as per [Fast DDS CONTRIBUTING guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md).
